.codacy.yml

```diff
@@ -15,5 +15,6 @@ exclude_paths:
   - 'deployments/**'
   - 'tests/acceptance/expected-failures-*.md'
   - 'tests/acceptance/features/bootstrap/**'
+  - 'tests/TestHelpers/**'
```

Comment:
I excluded `tests/TestHelpers` from codacy because I saw that the `bootstrap` folder has already been excluded.
Someone, please provide your opinion